biggraphite/drivers/elasticsearch.py

```diff
@@ -699,15 +714,15 @@ def map(
         for i, metric in enumerate(metrics):
             callback(metric, i, total)
 
-    def __touch_metadata_on_need(self, metric):
-        if not metric.updated_on:
+    def __touch_metadata_on_need(self, document):
```

Comment:
What do you think about renaming this `__maybe_set_updated_on` and same for the `read_on` methods ? That would be less confusing (potentially, we could do that in cassandra.py too)

Reply:
Let's do this in another refactoring PR